packages/dds/tree/src/test/feature-libraries/chunked-forest/codec/compressedEncode.spec.ts

```diff
@@ -125,6 +125,7 @@ describe("compressedEncode", () => {
 						anyFieldEncoder,
 					fieldKinds,
```

Comment:
This file is where unit tests for code in compressedEncode below. You added IncrementalChunkShape to compressedEncode.ts, so it should have a test suite here.

Reply:
Same for incrementalFieldEncoder

Reply:
Added tests for these.